### DISCUSSION TITLE
[KtonDAO Temp Check]Shorten Maximum KtonDAO Ring-Deposit Locking Period to 12 Months

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1603] [General]
## Background
The control of KTON’s issuance and economic model has been transferred to the Deposit contract, which is supposed to be under the ownership of KtonDAO (reference: [runtime/crab/src/migration.rs#L58-L79](https://github.com/darwinia-network/darwinia/blob/e9c5fad5b1345dcb3d3576e2e59384aab467d045/runtime/crab/src/migration.rs#L58-L79)). This transition grants KtonDAO complete autonomy over KTON's monetary policy and token economy, including the ability to independently govern future changes in issuance mechanisms.

## Motivation
This proposal is driven by two key motivations:

1. **Reducing Locking Duration to Minimize User Expectation Errors**:  
   Currently, the maximum Ring-Deposit locking period is 36 months. KTON rewards are issued and distributed at the time of locking, which can lead users to incorrectly assume that current KTON staking rewards will be sustained over the next three years. As RingDAO is actively researching improvements to the token economy (reference: [RingDAO Token Economy Discussion](https://github.com/orgs/ringecosystem/discussions/4)), these returns may fluctuate. A 36-month locking period risks users overestimating future returns. Reducing the maximum locking period to 12 months would allow users to make more accurate assessments of future returns, thereby avoiding unrealistic expectations. This adjustment also reduces resistance to forthcoming economic reforms from both KtonDAO and RingDAO, allowing for more agile monetary policy changes.

2. **Controlling KTON Issuance and Increasing Market Appeal**:  
   A shorter locking period helps to limit KTON's total issuance, preventing excessive supply that could dilute the token's value. By reducing the maximum locking period, KtonDAO gains greater control over KTON’s supply, increasing the scarcity and desirability of existing tokens. This would lead to a more stable and predictable value proposition for KTON holders, enhancing its overall market appeal.

## Proposed Change
Reduce the current maximum Ring-Deposit locking period from 36 months to 12 months under KtonDAO governance.

## Expected Impact

1. **New Ring-Deposit User**:  
   New Ring-Deposit Users will be guided(and limited) to lock over a shorter period, reducing the risks associated with long-term market and policy changes. This shorter locking period will enable users to better adapt to potential economic reforms implemented by KtonDAO and RingDAO.

2. **Token Economy**:  
   Shortening the locking period will also decrease the overall issuance of KTON, helping to better control the token's supply. This will maintain the growth potential of KTON’s value, ensuring a healthier and more stable economic model.

## Conclusion
This proposal seeks to improve the accuracy of reward expectations for Ring-Deposit users by reducing locking period, mitigating risks related to long-term locking, and enhancing the scarcity and attractiveness of KTON tokens. By doing so, KtonDAO will foster a healthier token economy and provide greater flexibility for future economic reforms. This change will reduce user uncertainty regarding long-term locks and establish a solid foundation for KTON’s sustainable development.

## Related Issues

https://github.com/darwinia-network/DIP-7/issues/22

**Comment by danairey** [comment]
Hi Hackfisher, the only use case of KTON token is ring rewards, if you reduce the ring rewards then whats the point of KTON? That would destroy the token unless you guys have some other use case for it?

**Reply by hackfisher** [reply]
> Hi Hackfisher, the only use case of KTON token is ring rewards, if you reduce the ring rewards then whats the point of KTON? That would destroy the token unless you guys have some other use case for it?

The only use case you mentioned is only the past, the use cases in future is up to the explore and expand of KtonDAO.

For KtonDAO, it has stable incomes from RING incentives and is building its own treasury, it only have 144,501 supply, and a very low market cap. Even if only targeting as a meme coin, it will have huge potential, not to say it can expand the commitment lock incentives model to other coins besides RING. I do not understand why KtonDAO should burden RingDAO for keeping high incentives permanently, if RingDAO have obstacle to build confidence in the long term in the ringnomics aspects, what’s the point of KTON value capture while keeping minting more supply?

**Comment by danairey** [comment]
The point of KTON was to get people to lock their tokens and not sell. So those that have not sold should not be punished by pulling the value. That's where its value comes from. I understand alot can be done with KTON but with that we need ideas and I'm not aware of any, that they could be. Ring has real tech, kton has nothing. Darwinia has never been a meme type project, only reason I have followed it this long is because its real value driven. If ring succeeds and gains users , well kton would fly just based on its value to ring. Taking away its partnership with ring, seems unfair.

**Reply by hackfisher** [reply]
Definitely not taking away its partnership with RING.

On the contrary, this is one way to strengthen its partnership with RingDAO to together make succeed factors, it's win-win IMO.


Technology cannot capture value alone by itself, real value comes to technology from users via applications, RingDAO's tech stack is stabilized and it will need more invest to focusing on applications and adoption in the next step, not to say the growing competition in this space. In the current market suitation, ringeconomics is one of the key cornerstones of change.

**Reply by hackfisher** [reply]
> The point of KTON was to get people to lock their tokens and not sell.

BTW, The market data already shown that most are selling instead of locking, one explanation could be the tokenomics.

Thinking in overall and macro, staking and locking itself contributes to starting liquidity in specific term, but there is no magic, rewards tokens by printing more tokens will not necessarily make it win in the broader token competition market.

**Reply by danairey** [reply]
Yes i like the fact that you thinking and trying things. The tokenomics has got zero to do with the problem, the problem is that people don't use the product and require ring. Users change the game. You keep doing your thing and good luck, this project deserves to fly; attaining users are the only way.

**Comment by sonoferin2000** [comment]
I do support reducing the lock period not sure if the number you have picked is the best one to fall on though, that is pretty drastic. but am on board with the idea of rrestraining the growth of supply.Why keep increasing the token supply when there’s no clear plan for its use in the meantime? Flooding the market with something that lacks clear utility doesn’t make sense. We need to develop more use cases for KTON to give it value beyond just simply staking,governance  and being sold,. I was initially drawn to the project because of the team’s original vision, where KTON holders were supposed to receive a portion of protocol fees,which never happened. I also think it is worth looking into adding a deflationary  aspect to KTON that would slowly reduce the supply, nothing drastic, it still needs to be tradable, but where it incentivises long term holding of the token, a burn and redistribute mechanism maybe,. but would be best if the reduction wasn done more subley then so forcefully. like the boiling frog analogy, could be done incrementally. drastic steps would be a huge shock. same thing with staking rewards, im im on board of reducing supply to stablise price, but has to be done by degrees rather then a sudden jolt. would give people time to adjust their strategy, its not fair that their hands are tied  while their tokens are bonded, they lose ability to adjust according to the new changes, and all at once would be to drastic of a change,and would cause resentment. but definitely slowing down the creation would have a positive effect, we should also consider gradual reduction from both sides, the creation and  a burn mechanism. that's my view. want to be clear of my view, and not just give broad blanket support to eliminating new issuance, that would be detrimental.

**Comment by cryptohustler101** [comment]
My idea is to bring KTON staking back into Evolution.Land, which initially drove its success. By updating the UI for easier interaction, we can gamify the experience and expand KTON's use cases. We could make KTON exclusive on DeFi platforms and introduce it as a token in other ecosystems. There’s a lot of potential for KTON that just hasn’t been tapped into yet.

**Comment by hackfisher** [comment]
Test version of the proposal on Crab: https://www.tally.xyz/gov/cktondao/draft/2444026673188832611

**Reply by hackfisher** [reply]
https://github.com/darwinia-network/DIP-7/pull/29/files

**Comment by sonoferin2000** [comment]
@hackfisher,  the more im thinking about this proposal,the more I am opposed to it.  those with big bags,will always be at the top,the little guy never has a shot going forward.people wont bother staking. i think this proposal is too drastic, and should be thought out in a more equitable way then what we have here.

**Reply by hackfisher** [reply]
> @hackfisher,  the more im thinking about this proposal,the more I am opposed to it.  those with big bags,will always be at the top,the little guy never has a shot going forward.people wont bother staking.

Lol, $KTON is only less than $2 on exchanges now, any one can buy it, it is a free market. Minting a large bunch of KTONs with low cost barriers at once for anyone also does not looks safe for government. (Think different)

**Reply by hackfisher** [reply]
Another approach to this is use veToken model, gain more voting power by committing more convictions (lock in gov). KtonStaking is now basically incentive for participating gov.


Gov utilities is naturally the fist utilities for KTON, DCDAO is more fitting for those want membership based voting. (Carbon vote or One vote for one member/person)

**Reply by sonoferin2000** [reply]
One alternative is to implement a tax or burn mechanism on transfers. If we opt for a tax, it could help fund the KTONDAO treasury, enabling us to make meaningful progress with this project. Whenever there’s interaction with any address outside the staking contract or the token owner, a small amount of KTON would be automatically burned, gradually reducing the supply.

Combining this with a scaled-down version of your proposed reduced locking period could address the issue from two angles without penalizing those who genuinely want to stake. Additionally, it allows for a decrease in the market supply of RING due to longer lock periods. This approach is much more balanced than what has been suggested.

 we should focus on onboarding projects to our network, requiring them to burn KTON for deployment, which aligns with the original plan. This would create a multifaceted strategy. It would also be beneficial to explore alternative uses for KTON beyond staking. Given that we have team members who communicate with other projects, what progress are they making in this area?

**Reply by hackfisher** [reply]
> you guys have people on the payroll that talks to other projects, what are they doing?

There are mainly two teams on the payroll including Itering and DCDAO, please note that these funds are from RingDAO via Darwinia Treasury and RingDAO Ecosystem Multisig. 

There is currently no person on the payroll from KtonDAO, as we all know there was no treasury accumulation for KtonDAO before execution of previous onchain proposal(There was no spent too), the KtonStaking frontend and smart contract support is contributed by the community/teams for free or via RingDAO ecosystem sponsored grants, the KtonDAO proposals activities are conducted by the KTON holders and voters， independent from RingDAO

**Comment by SasoLithops** [comment]
GM everyone!

Love the ongoing debate. The only way for KTON DAO to find more utility with KTON is to get its treasury funded. 
I agree with @cryptohustler101 here as KTON originated from Evolution Land and since Evolution Land is open source now, we can ask for grant on Polygon or Astar (especialy now on Soneium) to develop there. Grant could fund development of Evolution Land under KTON DAO. But we must first think well and finalize the proposal.
If you KTON DAO members mostly disagree with idea I still think main priority is to get KTON DAO treasury funded and not with RING only as that is suicide for both tokens atm.
To distance in a way from Ring I will vote yes. **As I think no one locks tokens anymore for a year let alone for two or three....**


Same goes for current proposal for CKTON which could be included in Crab continent or given another usecase and not be only dependant on Crab token.

**Comment by hackfisher** [comment]
https://www.tally.xyz/gov/ktondao/proposal/49317089087613543715610698832514380163941923554993253580612475622085059217596
